index.bs

```diff
@@ -573,12 +574,16 @@ authorizing an authenticator with which to complete the operation.
     to locate it later for use, and also contains metadata that can be used by the <a>[RP]</a> to assess the strength of the
     credential during registration.
 
-    The <dfn>credential</dfn> attribute contains an object whose attributes state the type of, and identifier for, the 
-    credential represented by {{ScopedCredentialInfo}}.
+    The <dfn>clientData</dfn> member contains the <a>clientDataJSON</a> (see [[#signature-format]]) passed to the authenticator
```

Comment:
well, the issue was that we were using "member" when we were talking about interface attributes, but now after the global change we're using "attribute" also when discussing dictionary members...     ( interfaces have attributes and dictionaries have members, go figure...)